src/app/app.module.ts

```diff
@@ -6,11 +6,20 @@ import { AppComponent } from './app.component'
 import { HomeComponent } from './home/home.component'
 import { RouterLink, RouterLinkActive, RouterOutlet, provideRouter } from '@angular/router'
 import routerConfig from './app.routes'
+import { NgbModule } from '@ng-bootstrap/ng-bootstrap'
+import { PropertyManagementComponent } from './property-management/property-management.component'
```

Comment:
why is this component being added here (PropertyManagementComponent) ? Is it needed?

Reply:
Done.